example/main.go

```diff
@@ -80,34 +80,26 @@ func main() {
 	time.Sleep(time.Hour)
 }
 
-func generateTLSConfig() (*tls.Config, error) {
-	key, err := rsa.GenerateKey(rand.Reader, 2048)
+// Setup a bare-bones TLS config for the server
+func generateTLSConfig() *tls.Config {
```

Comment:
Why change the signature of this function?

Reply:
The same: I took cert generation function from quic-go repo